server/database/config/build.sql

```diff
@@ -0,0 +1,47 @@
+BEGIN;
+
+DROP TABLE IF EXISTS admin, cohort, student, project, studentProject CASCADE;
+
+CREATE TABLE admin (
+    id SERIAL PRIMARY KEY,
+    username VARCHAR(100) NOT NULL,
+    password VARCHAR(100) NOT NULL
+);
+
+CREATE TABLE cohort (
+    id SERIAL PRIMARY KEY,
+    name VARCHAR(100) NOT NULL,
+    description TEXT NOT NULL,
+    imgUrl TEXT NOT NULL,
+    githubLink TEXT NOT NULL
+);
+
+CREATE TABLE student (
+    id SERIAL PRIMARY KEY,
+    name VARCHAR(100) NOT NULL,
+    email VARCHAR(200) NOT NULL UNIQUE,
+    imgUrl TEXT NOT NULL,
+    githubLink TEXT NOT NULL,
+    cohortId INTEGER REFERENCES cohort(id) ON UPDATE CASCADE
```

Comment:
add the on delete cascade  for all references in the tables so it doesn't make any errors if something  deleted 
```suggestion
    cohortId INTEGER REFERENCES cohort(id) ON UPDATE CASCADE ON DELETE CASCADE
```